backward.hpp

```diff
@@ -24,6 +24,8 @@
 #ifndef H_6B9572DA_A64B_49E6_B234_051480991C89
 #define H_6B9572DA_A64B_49E6_B234_051480991C89
 
+#include "backward_config.hpp"
```

Comment:
Using cp to copy around headers is mostly a bad idea. If you do so, you need to
manually check if there were updates / bugfixes. Basically this ends in a dozen 
of packages all including a different version of the library.

Installing the library and including it then makes sure, that all packages use a 
up to date version.

Anyway, one thing one could do is, to guard the include with the STACK_DETAILS_BFD & co
defines, so that it falls back to the old behaviour if the defines are present